demo-project/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.kedro]
 package_name = "demo_project"
 project_name = "modular-spaceflights"
-project_version = "0.18.4"
+kedro_init_version = "0.18.4"
```

Comment:
Dataset factories were only introduced in `0.18.12` so this should be bumped.

Reply:
Good catch!

Reply:
Good catch indeed ! :+1: The `kedro_init_version` has been bumped following this suggestion